src/lib/pinata/pinata.ts

```diff
@@ -35,16 +35,26 @@ export default function usePinata() {
     });
     data.append('pinataOptions', pinataOptions);
 
-    const res = await axios.post(url, data, {
+    const additionalParams = {
       maxBodyLength: 'Infinity' as any, // this is needed to prevent axios from erroring out with large files
       headers: {
         // @ts-ignore
         'Content-Type': `multipart/form-data; boundary=${data._boundary}`,
         pinata_api_key: apiKey,
         pinata_secret_api_key: apiSecret,
       },
-    });
+    }
+    
+    var res = {'data': {'IpfsHash': ''}};
+
+    try {
+        res = await axios.post(url, data, additionalParams);
```

Comment:
do we need to be making post requests for pinlist?

Reply:
nope - this was me thinking this codebase is shared with gmnh service - i reset this to original setup and will remove unused code in a future pr